utils/i18n.ts

```diff
@@ -14,3 +14,10 @@ export async function activate(locale: string): Promise<void> {
   i18n.load(locale, messages)
   i18n.activate(locale)
 }
+
+export async function activateAndSetCookie(locale: string): Promise<void> {
+  const { messages } = await import(`../locale/${locale}/messages.js`)
+  i18n.load(locale, messages)
+  i18n.activate(locale)
+  document.cookie = `current_locale=${locale}`
```

Comment:
I think We should not use `document.cookie`. It can cause Security Issues later.

Reply:
old site uses cookies for saving locale

Reply:
Can you please tell me why old site uses cookies for saving locale?